### PR TITLE
Added integer identifiers saving to a model in Fluent

### PR DESCRIPTION
**Pull request by Joannis**

### CODE REVIEW

File: Sources/FluentBenchmark/Pet.swift [hunk 1]
```diff
@@ -47,14 +47,6 @@ public final class Pet<D: Database>: Model {
         self.name = name
         self.ownerID = ownerID
     }
-
-    /// See Encodable.encode
-    public func encode(to encoder: Encoder) throws {
```
Comment (tanner0101) [inline Sources/FluentBenchmark/Pet.swift]: we should keep these if possible. they ensure that coding container api is working properly

File: Sources/FluentBenchmark/BenchmarkIntegerIdentifier.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Async
+import Dispatch
+import Fluent
+import Foundation
+
+extension Benchmarker where Database.Connection: SchemaSupporting {
+    public func benchmarkAutoIncrement() throws {
```
Comment (tanner0101) [inline Sources/FluentBenchmark/BenchmarkIntegerIdentifier.swift]: we should separate this out into a test that doesn't require schema supporting dbs like the rest of the tests

**Comment by tanner0101** [comment]
I _did_ implement this! But then accidentally deleted it when i was debugging the Future awaiters retain cycle. xD

I will reimplement on my form-url PR.

It should look something more like this:

```swift
            return try model.willCreate(on: conn)
                .then {
                    self.run { model in
                        try model.parseID(from: conn)
                    }
                }
                .then { try model.didCreate(on: conn) }
```

**Comment by Joannis** [comment]
To be reimplemented by @tanner0101

**Pull request closed without merging**